### DISCUSSION TITLE
矽谷大裁員

### DISCUSSION THREAD

**Discussion by cindy5D55** [1101-GS4719-C/Python-Community-GS4719-C#1483] [General]
截至5月25日，矽谷科技公司的裁員總數達到19萬9077人，已經逼近20萬大關，對於這些裁員行動，從企業和員工的角度來看，都有著各自不同的考量和理由。

不過，無論是從企業還是員工的角度來看，矽谷大裁員都反映出了科技行業目前所面臨的困境和挑戰。科技行業發展迅速，但同時也面臨著產業競爭激烈、技術更新快等問題，這些問題都對企業的發展和員工的工作產生了影響。

對於這些問題，我認為如果是從事相關行業不免會對此產生懷疑，害怕出社會後科技業的前景與現在有所不同，但若是有過硬的專業技能與學習能力，相信也可以面對此產業的變革。

https://www.bnext.com.tw/article/73845/2023-jan-tech-layoffs

**Comment by ashleyy-huang** [comment]
經常能夠看到大公司抑或是小公司進行裁員，雖然他們會徵各種不同領域的人才，但我覺得未來就算是畢業了我們還是要繼續學習新東西甚至不是本科系的也要學，持續進步向老闆證明能力，才不會成為被裁員的其中一個人。
